test/lib/views/eventloop-view.spec.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
-var expect = require("chai").expect;
+var chai = require("chai");
 var sinon = require("sinon");
+var sinonChai = require("sinon-chai");
+var expect = chai.expect;
+chai.use(sinonChai);
```

Comment:
GLOBAL: This shouldn't be necessary -- this boilerplate is already taken care of in `setup.js` https://github.com/FormidableLabs/nodejs-dashboard/blob/master/test/setup.js

I think maybe the gulp switch isn't using setup? At any rate go with how we already ran mocha from package.json scripts and should work fine.